lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -3189,6 +3189,8 @@ SPIRVValue *LLVMToSPIRVBase::transIntrinsicInst(IntrinsicInst *II,
   case Intrinsic::invariant_end:
   case Intrinsic::dbg_label:
   case Intrinsic::trap:
+  // llvm.instrprof.increment is not supported
```

Comment:
Please rearrange the lines so comment about `llvm.trap`  appears just before `case Intrinsic::trap`.